### ISSUE TITLE
ssl-related error attempting to make rpc calls from "admin" container

### ISSUE THREAD

**Issue by superquest** [PierreRochard/bitcoin-lightning-docker#19]
TLDR; The wsgi app is able to `import bitcoin.rpc` but a python shell within the `bitcoind_admin_website_service` container isn't.

-----

This is what I was trying to do
```
$ docker exec -it bitcoind_admin_website_service /bin/bash
$ python
>>> import python.rpc
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/bitcoin/rpc.py", line 48, in <module>
    from bitcoin.wallet import CBitcoinAddress, CBitcoinSecret
  File "/usr/local/lib/python3.6/site-packages/bitcoin/wallet.py", line 29, in <module>
    import bitcoin.core.key
  File "/usr/local/lib/python3.6/site-packages/bitcoin/core/key.py", line 33, in <module>
    _ssl = ctypes.cdll.LoadLibrary(ctypes.util.find_library('ssl') or 'libeay32')
  File "/usr/local/lib/python3.6/ctypes/__init__.py", line 426, in LoadLibrary
    return self._dlltype(name)
  File "/usr/local/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: Error loading shared library libeay32: No such file or directory
```
So it appears that somehow the admin container doesn't have an ssl library.

This is extremely odd because [admin.py is able to do this import just fine!](https://github.com/PierreRochard/bitcoin-lightning-docker/blob/master/services/admin_website/app/app/admin.py#L6).

The problem went away once I updated the admin container's Dockerfile to also install `openssl-dev`.

I'm leaving this issue here because other people might encounter it, and something more may be going on.
